docs/release/cut-sw-release.md

```diff
@@ -141,7 +141,11 @@ To test pre-release, you will be kicking off a manual VM universe test run from
 !!! note
     If there are failures, consult the release-manager before proceeding.
 
-### Step 3: Regenerate the build repositories
+### Step 3: Test Pre-Release on the Madison ITB site
+
+Test the pre-release on the Madison ITB by following the [ITB pre-release testing instructions](itb-testing/).
```

Comment:
What about people that don't have access to the Madison ITB?

Reply:
I think that everyone that can release will have access to the Madison ITB. Two ways to accomplish this. 1. Only Madison folks release. 2. Give access to remote team members.